sycl/source/detail/global_handler.cpp

```diff
@@ -79,6 +122,13 @@ GlobalHandler *&GlobalHandler::getInstancePtr() {
 GlobalHandler &GlobalHandler::instance() {
   GlobalHandler *RTGlobalObjHandler = GlobalHandler::getInstancePtr();
   assert(RTGlobalObjHandler && "Handler must not be deallocated earlier");
+
```

Comment:
InitXPTIStuff is called independently on xptiTryToEnable method. It is called when we query global handler in runtime. So it would be called even if xpti not enabled.

Reply:
Ok, sounds good!